dts/msc-sm2-imx6x.dtsi

```diff
@@ -589,25 +589,6 @@
 		compatible = "ti,tmp103";
 		reg = <0x71>;
 	};
-	
```

Comment:
That's a new rule to me. And a bit of a silly one, because this is the format:
"EMP-XXX: nnnnnnnnnnn. This contributes to EMP-XXX.."
and 11 n's is way to short to write anything useful.

Reply:
General git best practices see: https://chris.beams.io/posts/git-commit/ 
Anyway before we make the work final and merge to master we should take care of the git Hygiene. 

- 50 char max short description
- Long description
- Jira links